### PR TITLE
ADD documentation for nosignal rules

### PR DESCRIPTION
**Pull request by fgalan**
Based in https://github.com/telefonicaid/perseo-fe/blob/master/documentation/models.md

### CODE REVIEW

File: documentation/plain_rules.md [hunk 1]
```diff
@@ -608,6 +628,17 @@ pre-provisioned application associated to the twitter user.
...
+
+The structure of the `nosignal` field for no signal rules is as follows:
+
+-   checkInterval: _optional_ or _mandatory_ ???,time in minutes for checking the attribute
+-   attribute: _optional_ or _mandatory_ ???, attribute for watch
+-   reportInterval: _optional_ or _mandatory_ ???, time in seconds to see an entity as silent
+-   id: _optional_ or _mandatory_ ???, of the entity to watch
+-   idRegexp: _optional_ or _mandatory_ ???, regular expression to match entities by ID
+-   type: _optional_ or _mandatory_ ???,: type of entities to watch
+
```
[the remark this answers is not in the document]
Comment (fgalan) [reply documentation/plain_rules.md]: I guess that id and idRegexp should be mutually exclusive (I mean, you can use one or the other but not both at the same time), but not sure if the code is enforcing that...

File: documentation/plain_rules.md [hunk 2]
```diff
@@ -92,6 +125,17 @@ Please, be carefull with using non-ascii characters in the EPL syntax. It will p
 information on how to scape characters at
 [Esper site](http://esper.espertech.com/release-6.1.0/esper-reference/html/event_representation.html#eventrep-properties-escaping)
 
+## No signal conditions
+
+The no signal condition is specified in the `nosignal` configuration element, which is an object with the following fields:
+
+-   checkInterval: _optional_ or _mandatory_ ???,time in minutes for checking the attribute
```
Comment (manucarrace) [inline documentation/plain_rules.md]: checkInterval is recommended to be at least double of reportInterval. But we should warn that a very demanding value of checkInterval could impact on performance.

File: documentation/plain_rules.md [hunk 3]
```diff
@@ -608,6 +628,17 @@ pre-provisioned application associated to the twitter user.
 
 The `template` field performs [string substitution](#string-substitution-syntax).
 
+## `nosignal` field
+
+The structure of the `nosignal` field for no signal rules is as follows:
+
+-   checkInterval: _optional_ or _mandatory_ ???,time in minutes for checking the attribute
```
[the remark this answers is not in the document]
Comment (fgalan) [reply documentation/plain_rules.md]: Fixed in e2f779d60800ceaa4e2fe119fe7c1a99449dc6a1

File: documentation/plain_rules.md [hunk 4]
```diff
@@ -608,6 +628,17 @@ pre-provisioned application associated to the twitter user.
...
+-   attribute: _optional_ or _mandatory_ ???, attribute for watch
```
[the remark this answers is not in the document]
Comment (fgalan) [reply documentation/plain_rules.md]: Fixed in e2f779d60800ceaa4e2fe119fe7c1a99449dc6a1

File: documentation/plain_rules.md [hunk 5]
```diff
@@ -608,6 +628,17 @@ pre-provisioned application associated to the twitter user.
...
+-   reportInterval: _optional_ or _mandatory_ ???, time in seconds to see an entity as silent
```
[the remark this answers is not in the document]
Comment (fgalan) [reply documentation/plain_rules.md]: Fixed in e2f779d60800ceaa4e2fe119fe7c1a99449dc6a1

File: documentation/plain_rules.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (fgalan) [reply documentation/plain_rules.md]: > I guess that id and idRegexp should be mutually exclusive (I mean, you can use one or the other but not both at the same time), but not sure if the code is enforcing that...

It seems the code doesn't enforce that but, given it is common sense from my point of view :), I have changed the text to tell so.

Fixed in e2f779d

File: documentation/plain_rules.md (hunk 2 above)
Comment (fgalan) [reply documentation/plain_rules.md]: Fixed in 
e2f779d

**Pull request merged**